.github/workflows/create-release-v2.yaml

```diff
@@ -0,0 +1,183 @@
+                        "scan_mitre",
+                        "scan_with_exceptions",
+                        "scan_repository",
+                        "scan_local_file",
+                        "scan_local_glob_files",
+                        "scan_nsa_and_submit_to_backend",
+                        "scan_mitre_and_submit_to_backend",
+                        "scan_local_repository_and_submit_to_backend",
+                        "scan_repository_from_url_and_submit_to_backend",
+                        "host_scanner",
+                        "scan_local_list_of_files",
+                        "scan_compliance_score"
+                      ]'
+      DOWNLOAD_ARTIFACT_PATH: ${{ needs.build-and-rego-test.outputs.REGO_ARTIFACT_PATH }}
+    secrets: inherit
+
+  # start release process
+  create-new-tag-and-release:
+    needs: [ks-and-rego-test]
+    if: ${{ (always() && (contains(needs.*.result, 'success')) && !(contains(needs.*.result, 'skipped')) && !(contains(needs.*.result, 'failure')) && !(contains(needs.*.result, 'cancelled'))) }}
+    name: create release and upload assets
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        name: Checkout repository
+
+      - name: 'Generate Release Tag'
+        id: generate_tag
+        uses: kubescape/workflows/.github/actions/tag-action@main
+        with:
+          ORIGINAL_TAG: ${{ github.ref_name }}
+          SUB_STRING: "-rc."
+      
+      # Create and push the full version tag (e.g., v2.0.1)
+      - name: Create and Push Full Tag
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: ${{ steps.generate_tag.outputs.NEW_TAG }}
+          force_push_tag: false
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Generate Short Tag
+        id: short_tag
+        run: |
+          SHORT_TAG=$(echo "${{ steps.generate_tag.outputs.NEW_TAG }}" | grep -oP '^v\d+')
+          echo "Short tag: $SHORT_TAG"
+          echo "SHORT_TAG=$SHORT_TAG" >> $GITHUB_ENV
+
+      - name: Force Push Short Tag
+        uses: rickstaa/action-create-tag@v1
+        with:
+          tag: ${{ env.SHORT_TAG }}
+          force_push_tag: true
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - uses: actions/download-artifact@9bc31d5ccc31df68ecc42ccf4149144866c47d8a # ratchet:actions/download-artifact@v3.0.2
+        id: download-artifact
+        with:
+          name: ${{ env.REGO_ARTIFACT_KEY_NAME }}
+          path: ${{ env.REGO_ARTIFACT_PATH }}
+
+      - name: Create or Update Release and Upload Assets
+        uses: softprops/action-gh-release@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          tag_name: ${{ env.SHORT_TAG }} 
+          name: ${{ env.SHORT_TAG }}
+          body: "Automated release for ${{ env.SHORT_TAG}}"
+          files: ${{ env.REGO_ARTIFACT_PATH }}/*
+          draft: false
+          fail_on_unmatched_files: true
+          prerelease: false
+          make_latest: "false"
```

Comment:
Is this a string value?

Reply:
[yes](https://github.com/softprops/action-gh-release/blob/9d7c94cfd0a1f3ed45544c887983e9fa900f0564/README.md?plain=1#L186)